src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.File.Create.cs

```diff
@@ -10,7 +10,7 @@ namespace System.IO.Tests
 {
     public class File_Create_Tests : FileSystemWatcherTest
     {
-        [Fact]
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsSubsystemForLinux))] // https://github.com/Microsoft/BashOnWindows/issues/216
```

Comment:
For all of these "linked" to issues in another repo, how will we know when to come back and re-enable these?  Do we have an issue tracking in the corefx repo?  Is there a mention on the relevant issues for someone to notify us? Etc.